urls.py

```diff
@@ -84,7 +84,8 @@
     )),
     url(r'^zapier/', include('corehq.apps.zapier.urls')),
     url(r'^zipline/', include('custom.zipline.urls')),
-    url(r'^remote_link/', include('corehq.apps.linked_domain.urls'))
+    url(r'^remote_link/', include('corehq.apps.linked_domain.urls')),
+    url(r'^translations/', include('corehq.apps.translations.urls')),
```

Comment:
Was this committed by mistake?  (asking because it doesn't fit with the rest of the commit)  If it's intentional, I think the translations urls are referenced in two places.

Reply:
Was by mistake in an earlier commit which was then removed but now its expected change.